flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/agg/WindowAggregateTest.scala

```diff
@@ -683,4 +682,165 @@ class WindowAggregateTest extends TableTestBase {
       "must be an integral multiple of step, but got maxSize 3600000 ms and step 1500000 ms")
     util.verifyExplain(sql)
   }
+
+  @Test
+  def testCantMergeWindowTVF_GroupingSetsWithoutWindowStartEnd(): Unit = {
```

Comment:
Should be `testCantTranslateToWindowAgg_` because the group keys don't contain both window_start and window_end, so there is no `WindowAggregate`, but `GroupAggregate`.